extensions/adobe/experience/decisioning/proposition-attribution-detail.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/proposition-attribution-detail",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Proposition Acceptance Details",
+  "type": "object",
+  "meta:abstract": true,
+  "meta:extensible": true,
+  "description": "A decision response is used to link an experience event that is a response to a prior decision/proposition events. Each decision output has a unique ID that can be echo'ed back with an experience event. The decision is assumed to have contributed to the experience event and is now attributed with by it.",
+  "definitions": {
+    "proposition-attribution-detail-datatype": {
```

Comment:
We don't put names like "class", "mixin", "datatype" in the schema/definition names. Please remove "-datatype". The same applies to the definition name in the mixin as well.

Reply:
All schemas in this namespace use this convention, mostly for internal documentation help. None of the JSON definition names will contribute to the composed schema, they will be entirely invisible.
How necessary is it that this is being changed now. It would cause an inconsistency with the other files in this namespace

Reply:
Agree please leave them for this branch I find them helpful and non-intrusive or impacting